src/routes/__layout.svelte

```diff
@@ -1,42 +1,30 @@
 <script lang="ts">
   import '../app.css';
-</script>
-
-<main>
-  <slot />
-</main>
-
-<footer>
-  <p>visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to learn SvelteKit</p>
-</footer>
+  import AudioControls from 'src/features/playback-controls/ui/controls.svelte';
+  import { audio, setAudio } from 'src/entities/audio/model/audio-instance';
+  import { onMount } from 'svelte';
 
-<style>
-  main {
-    flex: 1;
-    display: flex;
-    flex-direction: column;
-    padding: 1rem;
-    width: 100%;
-    max-width: 1024px;
-    margin: 0 auto;
-    box-sizing: border-box;
-  }
-
-  footer {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    padding: 40px;
-  }
+  onMount(() => {
+    if (!$audio) setAudio(new Audio());
+  });
+</script>
 
-  footer a {
-    font-weight: bold;
-  }
+{#if $audio}
+  <main class="relative margin-auto h-full w-full max-w-md p-4 bg-slate-900 text-white">
+    <slot />
 
-  @media (min-width: 480px) {
-    footer {
-      padding: 40px 0;
-    }
-  }
-</style>
+    <div
```

Comment:
praise: wow, you actually implemented the bottom bar. That was supposed to be in the upcoming sprints, but that's cool.